docs/tutorials/contracts/nfts/minting-nfts.md

```diff
@@ -0,0 +1,356 @@
+
+### Testing the contract
+
+Written in the smart contract there are pre-written tests that you can run. Run the following command in your terminal to perform these simple tests to verify that your contract code is working.
+
+```bash
+cargo test -- --nocapture
+```
+
+> **Note:** the more complex simulation tests aren't performed with this command but we can find them in `tests/sim`.
+
+## Using the NFT contract
+
+Now that you have successfully built and tested the NFT smart contract, you're ready to [deploy it](#deploying-the-contract)
+and start using it mint your NFTs.
+
+### Deploying the contract
+
+This smart contract will be deployed to your NEAR account. Because NEAR allows the ability to upgrade contracts on the same account, initialization functions must be cleared.
+
+> **Note:** If you'd like to run this example on a NEAR account that has had prior contracts deployed, please use the `near-cli` command `near delete` and then recreate it in Wallet. To create (or recreate) an account, please follow the directions in [Test Wallet](https://wallet.testnet.near.org) or ([NEAR Wallet](https://wallet.near.org/) if we're using `mainnet`).
+
+Log in to your newly created account with `near-cli` by running the following command in your terminal.
+
+```bash
+near login
+```
+
+To make this tutorial easier to copy/paste, we're going to set an environment variable for your account ID. In the command below, replace `YOUR_ACCOUNT_NAME` with the account name you just logged in with including the `.testnet` (or `.near` for `mainnet`):
+
+```bash
+export ID=YOUR_ACCOUNT_NAME
+```
+
+Test that the environment variable is set correctly by running:
+
+```bash
+echo $ID
+```
+
+Verify that the correct account ID is printed in the terminal. If everything looks correct you can now deploy your contract. 
+In the root of your NFT project run the following command to deploy your smart contract.
+
+```bash
+near deploy --wasmFile res/non_fungible_token.wasm --accountId $ID
+```
+
+<details>
+<summary>Example response: </summary>
+<p>
+
+```bash
+Starting deployment. Account id: ex-1.testnet, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, file: res/non_fungible_token.wasm
+Transaction Id E1AoeTjvuNbDDdNS9SqKfoWiZT95keFrRUmsB65fVZ52
+To see the transaction in the transaction explorer, please open this url in your browser
+https://explorer.testnet.near.org/transactions/E1AoeTjvuNbDDdNS9SqKfoWiZT95keFrRUmsB65fVZ52
+Done deploying to ex-1.testnet
+```
+
+</p>
+</details>
+
+> **Note:** For `mainnet` you will need to prepend your command with `NEAR_ENV=mainnet`. [See here](/docs/tools/near-cli#network-selection) for more information.
+
+### Minting your NFTs
+
+The NFT contract should be initialized before usage, but for now we'll initialize with the default metadata:
```

Comment:
Maybe we should speak on initializing contracts and why we do this. Also, include a tip that reminds people that they will need to delete their account if they have already deployed a contract.

Reply:
Added some notes about contract initialization. I kept the notes about re-using/deleting an account in the `Deploying the contract` section.